pipelinewise/cli/tap_properties.py

```diff
@@ -51,139 +55,133 @@ def generate_tap_s3_csv_to_table_mappings(tap):
+            'default_data_flattening_max_level': 10
         },
-        'tap_stream_id_pattern': '{{table_name}}',
-        'tap_stream_name_pattern': '{{table_name}}',
-        'tap_catalog_argument': '--properties',
-        'default_replication_method': 'INCREMENTAL',
-        'default_data_flattening_max_level': 0
-    },
-
-    'tap-s3-csv': {
-        'tap_config_extras': {
-            'tables': generate_tap_s3_csv_to_table_mappings(tap)
+        'tap-adwords': {
+            'tap_config_extras': {},
+            'tap_stream_id_pattern': '{{table_name}}',
+            'tap_stream_name_pattern': '{{table_name}}',
+            'tap_catalog_argument': '--catalog',
+            'default_replication_method': 'INCREMENTAL',
+            'default_data_flattening_max_level': 0
         },
-        'tap_stream_id_pattern': '{{table_name}}',
-        'tap_stream_name_pattern': '{{table_name}}',
-        'tap_catalog_argument': '--properties',
-        'default_replication_method': 'INCREMENTAL',
-        'default_data_flattening_max_level': 0
-    },
-
-    'tap-snowflake': {
-        'tap_config_extras': {
-            # PipelineWise doesn't support replicating from multiple
-            # databases by one tap but tap-postgres does.
-            #
-            # To avoid problems of loading two tables with the same name
-            # but from differnet dbs we force tap-postgres to filter only
-            # the db that's in scope
-            'filter_dbs': tap.get('db_conn', {}).get('dbname') if tap else None
+        'tap-jira': {
+            'tap_config_extras': {
+                'user_agent': 'PipelineWise - Tap Jira'
+            },
+            'tap_stream_id_pattern': '{{table_name}}',
+            'tap_stream_name_pattern': '{{table_name}}',
+            'tap_catalog_argument': '--properties',
+            'default_replication_method': 'INCREMENTAL',
+            'default_data_flattening_max_level': 0
         },
-        'tap_stream_id_pattern': '{{database_name}}-{{schema_name}}-{{table_name}}',
-        'tap_stream_name_pattern': '{{schema_name}}-{{table_name}}',
-        'tap_catalog_argument': '--properties',
-        'default_replication_method': 'INCREMENTAL',
-        'default_data_flattening_max_level': 0
-    },
-
-    'tap-salesforce': {
-        'tap_config_extras': {
-            'select_fields_by_default': True
+        'tap-s3-csv': {
+            'tap_config_extras': {
+                'tables': generate_tap_s3_csv_to_table_mappings(tap)
+            },
+            'tap_stream_id_pattern': '{{table_name}}',
+            'tap_stream_name_pattern': '{{table_name}}',
+            'tap_catalog_argument': '--properties',
+            'default_replication_method': 'INCREMENTAL',
+            'default_data_flattening_max_level': 0
+        },
+        'tap-snowflake': {
+            'tap_config_extras': {
+                # PipelineWise doesn't support replicating from multiple
```

Comment:
Hey, I dont think this comment is relevant here.

Reply:
fixed by 092a761e802245ca4ea8f6200fffd0ee15c80864